packages/extras/src/lib/interactivity/setupInteractivity.ts

```diff
@@ -61,60 +58,46 @@ export const setupInteractivity = (state: State) => {
     }
   }
 
-  const enabled = memoize(state.enabled)
+  const enabled = memoize(context.enabled)
 
   const getHits = (): Intersection[] => {
-    const duplicates = new Set<string>()
+    if (!enabled) return []
```

Comment:
```suggestion
    if (!enabled.current) return []
```

Reply:
Ah! Good catch. Should have gone with an explicit comparison 🤦